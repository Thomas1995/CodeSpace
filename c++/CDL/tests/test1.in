Testam o figura inclusa in alta

Vreau un patrat in punctul (0,0) de latura 3
Vreau un patrat in punctul (1,1) de latura 1

Daca figura 0 intersecteaza figura 1
Vreau un poligon cu punctele (0,0), (3,3), (4,4)
Altfel
Vreau un poligon cu punctele (2,2), (3,3), (4,4)
Gata boss

Daca figura 1 intersecteaza figura 0
Vreau un poligon cu punctele (1,1), (3,3), (4,4)
Gata boss
