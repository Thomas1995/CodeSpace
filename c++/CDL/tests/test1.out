0 Patrat:
0 0
0 3
3 0
3 3
1 Patrat:
1 1
1 2
2 1
2 2
2 Poligon:
0 0
3 3
4 4
3 Poligon:
1 1
3 3
4 4
